core/config/entity.go

```diff
@@ -19,6 +19,8 @@ type SentinelConfig struct {
 		// Type indicates the classification of the service (e.g. web service, API gateway).
 		Type int32
 	}
+	// EnableMetricLog represents whether to enable metric monitor log
+	EnableMetricLog bool `yaml:"enableMetricLog"`
```

Comment:
Maybe it's not needed. Setting `sentinel.log.metric.flushIntervalSec` to 0 is enough.

Reply:
Make sense.
I will remove this config field.